templates/admin/map.tpl

```diff
@@ -1,22 +1,22 @@
 {if !empty($onlineMembers)}
-	<div class="widget widget-large" id="widget-website-visits">
-		<div class="widget-header"><i class="i-users"></i> {lang key='online_members_map'}
-			<ul class="nav nav-pills pull-right">
-				<li><a href="#" class="widget-toggle"><i class="i-chevron-up"></i></a></li>
-			</ul>
-		</div>
-		<div class="widget-content">
-			<div id="map-canvas" style="width: 100%; height:260px"></div>
-		</div>
-	</div>
-	<script type="text/javascript" src="http://maps.googleapis.com/maps/api/js?key=AIzaSyDbz0xfgjrFJjceGiGXtE4_JKX9buZGyOU&sensor=false"></script>
-	{ia_add_media files='js:_IA_URL_modules/member_map/js/admin/index'}
+    <div class="widget widget-large" id="widget-website-visits">
+        <div class="widget-header"><i class="i-users"></i> {lang key='online_members_map'}
+            <ul class="nav nav-pills pull-right">
+                <li><a href="#" class="widget-toggle"><i class="i-chevron-up"></i></a></li>
+            </ul>zz
+        </div>
+        <div class="widget-content">
+            <div id="map-canvas" style="width: 100%; height:260px"></div>
+        </div>
+    </div>
+    <script type="text/javascript" src="http://maps.googleapis.com/maps/api/js?key=AIzaSyDbz0xfgjrFJjceGiGXtE4_JKX9buZGyOU&sensor=false"></script>
```

Comment:
Looks like we have hardcoded key here. What's that? Please recheck. If it's mandatory, we should use it as a configuration.